client/src/components/NewReviewForm.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+
+const NewReviewForm = ({ postReview }) => {
+  const [newReview, setNewReview] = useState({
+    title: "",
+    rating: "",
+    content: "",
+  });
+
+  const handleInputChange = (event) => {
+    event.preventDefault();
+    setNewReview({
+      ...newReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postReview(newReview);
+    clearForm();
+  };
+
+  const clearForm = () => {
+    setNewReview({
+      title: "",
+      rating: "",
+      content: "",
+    });
+  };
+
+  return (
+    <div>
+      <h1>Add a Review</h1>
+      <form onSubmit={handleSubmit}>
+        <label>
+          Title:
+          <input type="text" name="title" onChange={handleInputChange} value={newReview.title} />
+        </label>
+        <label>
+          content:
+          <input
```

Comment:
As we expect more text in `content`, you should consider using `<textarea>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea